pkg/pgmodel/metrics/database/database.go

```diff
@@ -114,11 +114,17 @@ func (e *metricsEngineImpl) Run() error {
 func (e *metricsEngineImpl) Update() error {
 	batch := e.conn.NewBatch()
 	batchMetrics := []metricQueryWrap{}
+	now := time.Now()
```

Comment:
Nit: it would be more correct to grab `Now()` before checking customPoll config b/c we do healthCheck call.